shared/Microsoft.Extensions.Certificates.Generation.Sources/CertificateManager.cs

```diff
@@ -0,0 +1,414 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Runtime.InteropServices;
+using System.Security.Cryptography;
+using System.Security.Cryptography.X509Certificates;
+using System.Text;
+
+namespace Microsoft.Extensions.Certificates.Generation
+{
+    internal class CertificateManager
```

Comment:
Why is this shared source instead of a traditional library?

Reply:
We don't want to expose public API for this. It's a complete implementation detail